tests/tests_geomstats/test_stiefel.py

```diff
@@ -81,6 +85,18 @@ def test_random_uniform_shape(self):
 
         self.assertAllClose(gs.shape(result), (self.n, self.p))
 
+    @geomstats.tests.np_only
```

Comment:
We can remove this decorator, after the removal of the assignment it should work in three backends